app/src/main/res/xml/root_preferences.xml

```diff
@@ -28,6 +28,12 @@
             android:key="@string/setting_key_exclude_from_recent"
             android:title="@string/setting_exclude_from_recent"
             app:singleLineTitle="false" />
+
+        <SwitchPreferenceCompat
```

Comment:
Does this solve a problem you have? I'd like to avoid settings unless absolutely necessary.

Reply:
It's just offering the users to decide whether to enable translucent status bar. most applications have apply the translucent status bar. but I don't know if it should be default enabled for gotify. so I created a switcher in the settings.

Reply:
enabled:
![Screenshot_2023-05-01-18-09-00-865_com yaocf gotify](https://user-images.githubusercontent.com/28947794/235439056-17ffd41f-37b8-4699-b65a-ed2553664f2b.jpg)
disabled:
![Screenshot_2023-05-01-18-09-09-378_com yaocf gotify](https://user-images.githubusercontent.com/28947794/235439093-3767d516-c0f3-43ce-8ae2-6eeacbc8be6e.jpg)

Reply:
by default(same as disabled), the status bar will be black. but, most of applications was not black.